lib/components/Modal.js

```diff
@@ -10,6 +10,18 @@ var Assign = require('lodash.assign');
 var SafeHTMLElement = ExecutionEnvironment.canUseDOM ? window.HTMLElement : {};
 var AppElement = ExecutionEnvironment.canUseDOM ? document.body : {appendChild: function() {}};
 
+function getParentElement(parentSelector) {
```

Comment:
I'm personally not so much of a fan of handling all these options here.  Lately I've felt like maybe just going the function route is the best.  That way the prop in all cases is just a function that returns a DOM node.  Mixed examples don't have to be created and you don't have any issues with querySelector not returning what you expect it to, etc.  Push the selecting of an element all out to userland rather than dealing with it in internal to the library.